app/models/like.rb

```diff
@@ -0,0 +1,4 @@
+class Like < ApplicationRecord
+  belongs_to :author, class_name: 'User'
+  belongs_to :post, counter_cache: :like_counter
```

Comment:
- [ ] Can you create a method that updates the likes counter for a post? It is a requirement of the project.
- [ ] **Optional**: Just as I suggested the optional above:  `after_save :update_comments_counter`

Reply:
As requested by the previous reviewer, please create a method to update the likes counter for the post in this file just like you created the comments counter. It is a requirement of the project.

- [ ] Optional

- As suggested by the previous reviewer, consider using rails active callback `after_save` for the method once you have created it. Doing this will save you a lot of headache later since you won't have to call this method everytime you create a like on the post.